hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheGetAllOperation.java

```diff
@@ -71,8 +71,9 @@ public int getId() {
     }
 
     @Override
-    public String toString() {
-        return "CacheGetAllOperation{" + "keys:" + keys.toString() + "expiryPolicy:" + expiryPolicy + '}';
+    protected void toString(StringBuilder sb) {
+        sb.append(", keys=").append(keys.toString());
+        sb.append(", expiryPolicy=").append(expiryPolicy);
     }
 
```

Comment:
The hierarchy looks like this here:
```
CacheGetAllOperation extends PartitionWideCacheOperation
PartitionWideCacheOperation extends AbstractNamedOperation
```

As I reckon, the `name` should be added to the toString's output, but will be not.
Shouldn't the `toString(StringBuilder sb)` call `super.toString(sb)` at the beginning to make sure that all classes up the hierachy tree could add their input too (like in serialization)?